include/odp/api/spec/traffic_mngr.h

```diff
@@ -278,6 +278,24 @@ typedef struct {
 	 * of the weights allowed at this level. */
 	uint32_t max_weight;
 
+	/** min_burst only has significane when the nodes at this level support
+	 * TM shaping, in which case it specifies the smallest value of
+	 * the shaper burst allowed at this level. Unit is in bits.
+	 *
+	 * @see odp_shaper_profile_params_t::commit_burst
+	 * @see odp_shaper_profile_params_t::peak_burst
+	 */
+	uint32_t min_burst;
```

Comment:
Now that packet mode was added, there probably should be similar capability fields for the max and min burst sized in packets. IOW, the capability in the units of bits does not mean anything in packet mode.

So, maybe there should be separate capability fields for bit and packet mode, e.g. something like this:

```
/** Minimum allowed value for odp_tm_shaper_params_t::commit_burst when
 *  odp_tm_shaper_params_t::packet_mode is true.
 */
uint32_t min_burst_packets;

/** Minimum allowed value for odp_tm_shaper_params_t::commit_burst when
 *  odp_tm_shaper_params_t::packet_mode is false.
 */
```

Note that the type in the comment should be odp_tm_shaper_params_t.